.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Can use the full .gitignore for Node.js from https://github.com/github/gitignore/blob/main/Node.gitignore